boards/boardsource/Lulu/kb.py

```diff
@@ -0,0 +1,71 @@
+import board
+
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.scanners import DiodeOrientation
+from kmk.scanners.encoder import RotaryioEncoder
+from kmk.scanners.keypad import MatrixScanner
+
+
+class KMKKeyboard(_KMKKeyboard):
+    def __init__(self):
+        # create and register the scanner
+        self.matrix =[MatrixScanner(
```

Comment:
Formatting here is funky in a way I suspect black should be complaining about (?)

drop this to a new line at the same indent level as its closing paren and the opening of `RotaryioEncoder`

Reply:
Black formatted that, so.... I hope not. Can do though.